torch/distributed/optim/apply_optimizer_in_backward.py

```diff
@@ -0,0 +1,76 @@
+from typing import Any, Dict, Iterable, Type
+
+import torch
+
+__all__ = []  # type: ignore[var-annotated]
+
+
+def _apply_optimizer_in_backward(
+    optimizer_class: Type[torch.optim.Optimizer],
+    params: Iterable[torch.nn.Parameter],
+    optimizer_kwargs: Dict[str, Any],
+) -> None:
+    """
+    Upon backwards(), parameters will fire the corresponding optimizer.
```

Comment:
nit: The backtick is a super nit, but the important thing is that the method is `backward()`.
```suggestion
    Upon ``backward()``, parameters will fire the corresponding optimizer.
```